src/pages/index.js

```diff
@@ -33,3 +110,38 @@ const IndexPage = () => {
 }
 
 export default IndexPage
+
+export const homeQuery = graphql`
+query {
+  hill: file(relativePath: { eq: "other/hill.jpg"}){
+    childImageSharp{
+      fluid(quality: 70, maxWidth: 1400){
```

Comment:
The image in the page maxes out at 1232px wide so shouldn't that be the `maxWidth` here?